config/igor.yml

```diff
@@ -12,10 +12,11 @@ jenkins:
 
 redis:
   connection: ${services.redis.connection:redis://localhost:6379}
+  
 
-services:
-  clouddriver:
-    baseUrl: ${services.clouddriver.baseUrl}
+# services:
```

Comment:
Maybe change this to a comment that looks something like
# Also depends on "services.clouddriver.baseUrl", and "services.echo.baseUrl"
# which are normally declared directly in spinnaker[-local].yml